src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -319,6 +324,12 @@ CryptoNative_SslCtxSetCertVerifyCallback(SSL_CTX* ctx, SslCtxSetCertVerifyCallba
     SSL_CTX_set_cert_verify_callback(ctx, callback, arg);
 }
 
+void ResetProtocolRestrictions(SSL_CTX* ctx)
```

Comment:
```suggestion
static void ResetProtocolRestrictions(SSL_CTX* ctx)
```

It doesn't need to be accessed outside this file, right?

Reply:
I always thought this is no-op in C but I think this will suppress the warning I was seeing

Reply:
Nah, it just means something completely different than in C#: This function is not callable outside of this file.